src/dns_private.tf

```diff
@@ -22,3 +22,11 @@ resource "azurerm_private_dns_a_record" "private_dns_a_record_db_nodo" {
   ttl                 = 60
   records             = var.dns_a_reconds_dbnodo_ips
 }
+
+resource "azurerm_private_dns_zone" "privatelink_redis_pagopa_proxy" {
```

Comment:
U have to link private dns zone with virtual network using virtual network link

example:
https://github.com/pagopa/selfcare-infra/blob/main/src/core/dns_private.tf#L64

Reply:
Thanks! Fixed in 1f43f52